scripts/mainnetGetAdditionalFactoryCollectionItemsRaribleNFT.cdc

```diff
@@ -101,21 +102,22 @@ pub fun fetchRaribleNFTs(user: String, collectionIDs: {String : [UInt64]}) : {St
 
 	let fetchingIDs = collectionIDs
 	for project in fetchingIDs.keys {
-		let returnedNFTs = getNFTs(ownerAddress: account!.address, ids: collectionIDs[project]!)
 
 		var collectionItems : [MetadataCollectionItem] = []
-		for nft in returnedNFTs {
-			if nft == nil {
+		for id in fetchingIDs[project]! {
+
+			if !FlowverseSocksIds.contains(id) {
 				continue
 			}
-			
+            
+			let image = "https://img.rarible.com/prod/video/upload/t_video_big/prod-itemAnimations/FLOW-A.01ab36aaf654a13e.RaribleNFT:15029/b1cedf3"
 			let item = MetadataCollectionItem(
-				id: nft!.id,
-				name: nft!.display!.name,
-				collection:"RaribleNFT" ,
-				subCollection: project, 
-				media: nft!.display!.thumbnail.uri(),
-				mediaType: "image/png",
+				id: id,
+				name: "Flowverse socks",
+				collection: "Rarible",
+				subCollection: "Flowverse socks", 
```

Comment:
Can you change this to "" and collection to FlowverseSocks, that is the name of the collection in v1